src/tests/flask/security_route_middleware_test.py

```diff
@@ -0,0 +1,49 @@
+import unittest
+from unittest import mock
+from injector import Injector
+from src.constants.http import HttpMethods
+from src.constants.oauth2 import Roles, Scopes
+from src.core.di_config import DiProviders
+from src.env_config import get_env_config_instance
+from src.flask.routes.security_route_middleware import setup_security_route_middleware
+from src.tests.flask.before_request_middleware_function import before_request_middleware
+from src.tests.utils.fixtures.endpoint_request_fixture import create_request
+from src.tests.utils.fixtures.token_fixture import build_user_access_token, build_expire_access_token
+
+
+class SecurityRouteMiddlewareTest(unittest.TestCase):
+    def setUp(self):
+        self.env_config = get_env_config_instance()
```

Comment:
I feel we are doing this too much:

```
self.env_config = get_env_config_instance()
self.ioc = Injector(DiProviders)
```

Feels like we can put this into a base class.

This is just a suggestion.

Reply:
Done. This way is more scalable.